branchLayoutApi/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -51,4 +54,49 @@ public IBranchLayout slideOut(String branchName) throws BranchLayoutException {
       return List.of(entry.withChildren(children.flatMap(child -> slideOut(child, entryToSlideOut))));
     }
   }
+
+  @Override
+  public IBranchLayout slideIn(String aboveBranchName, String branchName) throws BranchLayoutException {
+    var entry = findEntryByName(branchName).getOrNull();
+    if (entry != null) {
```

Comment:
So as to not create the impression that `entry` will correspond to `entry` param of the other `slideIn` method:
```suggestion
    if (findEntryByName(branchName).isDefined()) {
```